modules/ROOT/pages/apikit-odatav4-expand-tutorial.adoc

```diff
@@ -0,0 +1,162 @@
+= Implement `$expand` leveraging flow reuse
+
+Since version 1.3.0 the module supports automatic routing for ease of
+implementing the `$expand` system query option on most common cases. Usage of
+this feature requires a little bit of preparation and configuration but it
+allows easy reuse of already implemented flows when processing `$expand`
+requests.
+
+For more information on the system query option see http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_SystemQueryOptionexpand[the OASIS specification].
```

Comment:
For more information on the system query option, see http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_SystemQueryOptionexpand[the OASIS specification].

Reply:
Don't forget to format this link as external:

```
For more information on the system query option, see http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_SystemQueryOptionexpand[the OASIS specification^].

Reply:
Right you are! Thanks!